README.md

```diff
@@ -1,55 +1,37 @@
-# Droopler profile for Drupal 8 #
+# Droopler profile for Drupal
 <img src="https://droopler-demo.droptica.com/themes/custom/droopler_subtheme/logo.svg" width=300 alt="Droopler Logo" />
 
-## What is Droopler? ##
-Droopler is a Drupal 8 profile designed to build corporate websites. It's based on the latest frontend technologies, including Bootstrap 4. The maintainer of Droopler is [Droptica](https://www.droptica.com).
+## What is Droopler?
+Droopler is a Drupal 8/9 profile designed to build corporate websites. It's based on the latest frontend technologies, including Bootstrap 4. The maintainer of Droopler is [Droptica](https://www.droptica.com).
 
 * **Official website**: [droptica.com/droopler](https://www.droptica.com/droopler)
-* **Documentation**: [droptica.com/droopler/for-developers](https://www.droptica.com/droopler/for-developers/)
+* **Tutorials**: [droptica.com/droopler/tutorials](https://www.droptica.com/droopler/tutorials/)
 * **Demo**: [droopler-demo.droptica.com](https://droopler-demo.droptica.com)
 * **Composer template**: [github.com/droptica/droopler_project](https://github.com/droptica/droopler_project)
 * **Drupal.org project**: [drupal.org/project/droopler](https://www.drupal.org/project/droopler)
 
 For the latest news please subscribe to our [Facebook](https://www.facebook.com/Droopler/) and [Twitter](https://twitter.com/DrooplerCMS).
 
-## What's in this repository? ##
-This repository contains a Drupal profile. When you put it in your /profiles directory, the Drupal installer gets modified and installs base Droopler theme, some module dependencies, and demo content.
+## What's in this repository?
+This repository contains a Drupal profile. When you put it in the `/profiles/contrib/droopler` directory, the Drupal installer gets modified and installs base Droopler theme, some module dependencies, and demo content.
```

Comment:
contains Drupal czy Droopler profile?

Reply:
but Droopler **is** a Drupal profile :D

Reply:
> and installs base Droopler theme, some module dependencies, and demo content.

@grzegorz-pietrzak-droptica  but maybe we should mention here that there is an option to choose some "optional module and/or demo content" ?

Reply:
No tak, problem kwadratu i prostokąta  :D
Każdy Droopler jest Drupalem ale nie każdy Drupal drooplerem.
W kazdym razie tylko chcialem sie co do tego upenwić